pilot/pkg/xds/endpoint_builder.go

```diff
@@ -384,9 +386,12 @@ type mtlsChecker struct {
 	subsetPolicyMode map[string]*networkingapi.ClientTLSSettings_TLSmode
 	// the tlsMode of the root traffic policy if it's set
 	rootPolicyMode *networkingapi.ClientTLSSettings_TLSmode
+
+	// Indicates the cluster we're checking settings for doesn't care about mTLS settings on the client (DestinationRule).
+	passthroughMode bool
```

Comment:
This is confusing, i think you mean for east west gateway, the tls mode of from DR is ignorred

Reply:
`cluster` as in xDS cluster. The passthrough clusters don't actually apply client TLS settings since they don't terminate/originate any TLS.